### PR TITLE
fix `Override the 'getElementsByTagName' method of the "HTMLHtmlElement" prototype` (close #1918)

### LINKED ISSUE #1918: Override the 'getElementsByTagName' method of the "HTMLHtmlElement" prototype
Related to https://github.com/DevExpress/testcafe-hammerhead/issues/1834.
Reproduced on http://developers.facebook.com/docs/plugins/embedded-video-player/

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

File: src/client/sandbox/shadow-ui.ts [hunk 1]
```diff
@@ -325,16 +325,17 @@ export default class ShadowUI extends SandboxBase {
     }
 
     _overrideElementMethods (window) {
-        const bodyProto = window.HTMLBodyElement.prototype;
-        const headProto = window.HTMLHeadElement.prototype;
+        const elementProto = window.Element.prototype;
```
Comment (miherlosev) [inline src/client/sandbox/shadow-ui.ts]: const htmlProto = `window.HTMLHtmlElement.prototype`

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
ping

File: src/client/sandbox/native-methods.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 /*global Document, Window */
+const NATIVE_CODE_RE = /\[native code]/;
```
Comment (miherlosev) [inline src/client/sandbox/native-methods.ts]: Let's create a static method in the NativeMethods sandbox and use it everywhere:
```js
NativeMethods.isNativeCode(code)
```

**Pull request merged**